packages/core/src/new-api/deployer.ts

```diff
@@ -31,11 +31,11 @@ export class Deployer {
   private _moduleConstructor: ModuleConstructor;
   private _executionEngine: ExecutionEngine;
   private _transactionService: TransactionService;
-  private _artifactLoader: ArtifactResolver;
   private _strategy: ExecutionStrategy;
+  private _artifactResolver: ArtifactResolver;
 
   constructor(
-    options: { journal: Journal; artifactLoader: ArtifactResolver } & (
+    options: { journal: Journal; artifactResolver: ArtifactResolver } & (
```

Comment:
A refactor to make the name and property match. This constructor has a todo for a complete rewrite.